modulus/datapipes/climate/era5_hdf5.py

```diff
@@ -306,11 +323,17 @@ def _create_pipeline(self) -> dali.Pipeline:
 
             # Crop.
             h, w = self.img_shape
-            invar = invar[:, :h, :w]
+            if self.num_history == 0:
```

Comment:
Just curious, can you do:
```py
invar = invar[..., :h, :w]
```
instead of this `if` clause?

Reply:
Thanks, switched to this